.github/workflows/ci.yml

```
@@ -6,26 +6,36 @@ on:
   pull_request:
     branches: [main, develop]
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)